grpc-service/Dockerfile

```diff
@@ -2,5 +2,5 @@ FROM openjdk:8
 
 ADD target/grpc-service-1.0-SNAPSHOT.jar /home/app.jar
 ADD rockyou.tar.gz /home
-
+CMD tar xf /home/rockyou.tar.gz /home
```

Comment:
so weird .. it should be extracted automatically .. would you please send me a copy of the error?! are you using MAC?!